packages/playground/src/pages/Cards/Cards.js

```diff
@@ -5,16 +5,15 @@ import { handleSubmit, handleAdditionalDetails, handleError } from '../../handle
 import { amount, shopperLocale } from '../../config/commonConfig';
 import '../../../config/polyfills';
 import '../../style.scss';
-import { countryCode } from '../../config/commonConfig';
 
 getPaymentMethods({ amount, shopperLocale }).then(paymentMethodsResponse => {
     window.checkout = new AdyenCheckout({
         amount,
         clientKey: process.env.__CLIENT_KEY__,
         paymentMethodsResponse,
         locale: shopperLocale,
-        countryCode,
         environment: 'test',
+        //        environment: 'http://localhost:8080/checkoutshopper/',
```

Comment:
Can we remove this line?

Reply:
I don't know. We have it in other playground files. It's kind of handy to not have to keep typing it out

Reply:
removed